storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst

```diff
@@ -668,7 +668,23 @@ Options
    momentarily stop the replication replica thread, this may help the backup to
    succeed and you do not need to use this option.
 
-.. include:: ../.res/contents/option.no-version-check.txt
+.. option:: --no-version-check
+
+   This option disables the version check when you add the ``--no-version-check`` option when invoking *Percona XtraBackup* 
```

Comment:
when invoking *Percona XtraBackup* -> to invoke

Reply:
Rewritten